### PR TITLE
Update INSTALL.md updated CentOS8 install instructions

### PR DESCRIPTION
**Pull request by ricco386**
### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)
* [x] Code style is correct (run the [linters](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#code-style))

### CODE REVIEW

**Comment by ricco386** [comment]
Signed-off-by: Richard Kellner <richard.kellner@gmail.com>

**Comment by ricco386** [comment]
Not entirely sure what broke the build? I have just changed the documentation. Any one, any ideas?

File: INSTALL.md [hunk 1]
```diff
@@ -124,12 +124,22 @@ sudo pacman -S base-devel python python-pip \
 
 #### CentOS/Fedora
 
-Installing prerequisites on CentOS 7 or Fedora 25:
+Installing prerequisites on CentOS 8 or Fedora>26:
+
+```
+sudo dnf config-manager --set-enabled PowerTools  # lcms2-devel is in PowerTools repository
+sudo dnf install libtiff-devel libjpeg-devel libzip-devel freetype-devel \
+                 lcms2-devel libwebp-devel tcl-devel tk-devel redhat-rpm-config \
```
Comment (richvdh) [inline INSTALL.md]: I can't help but wonder what things like `lcms2-devel`, `tcl-devel`, etc are needed for. We don't seem to have similar requirements for debian.

I realise you've just copied this from the centos 7 instructions, but could I impose on you to confirm if these are really needed?

Comment (ricco386) [reply INSTALL.md]: sure, is it enough if I install via pip and start the synapse? Or do I need to do other tests?

Comment (richvdh) [reply INSTALL.md]: yeah I think if you can install via pip that's good enough.

Comment (ricco386) [reply INSTALL.md]: bare minimum to install is: `dnf install python3` then I have created virtual environment activated the pip and upgrade services.... afterwards installing synapse in Centos8 works fine.

I was able to generate config, activate virtual environments and run synapse without exception been displayed.

I guess the `cms2-devel`, `tcl-devel` could be removed.

**Comment by ricco386** [comment]
Since we have started discussion regarding requirements, I wonder shouldn't we change
`virtualenv -p python3 ~/synapse/env` into `python3 -m venv ~/synapse/env` since [venv](https://docs.python.org/3/library/venv.html#venv-def) is core python module since 3.3. The way it is currently written it has one extra requirement `python3-virtualenv` which seems to be unnecessary as that is 3rd party package.

I have tested on CentOS8 and it works fine to use python core  module venv.

**Comment by ricco386** [comment]
hey @richvdh anything else that should be done regarding this issue?

**Pull request merged**